publish_live.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ev
```

Comment:
You might want -x rather than -v, which echoes lines as they are _executed_ rather than read.